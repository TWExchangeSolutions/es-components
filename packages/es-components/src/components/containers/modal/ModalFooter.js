import styled from 'styled-components';

// Note: ModalFooter relies on a parent (Modal) with ThemeProvider wrapping it
const ModalFooter = styled.div`
  background-color: ${props => props.theme.colors.gray2};
  border-top: 1px solid ${props => props.theme.colors.gray5};
  padding: 15px;
  text-align: right;
`;

export default ModalFooter;
