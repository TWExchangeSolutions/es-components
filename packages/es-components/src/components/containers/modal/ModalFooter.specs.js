/* eslint-env jest */

import React from 'react';
import renderer from 'react-test-renderer';
import { ThemeProvider } from 'styled-components';
import viaTheme from 'es-components-via-theme';

import ModalFooter from './ModalFooter';

describe('modal footer component', () => {
  let instanceToRender;

  beforeEach(() => {
    instanceToRender = (
      <ThemeProvider theme={viaTheme}>
        <ModalFooter className="footer">This is the footer.</ModalFooter>
      </ThemeProvider>
    );
  });

  it('renders as expected', () => {
    const tree = renderer.create(instanceToRender).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
