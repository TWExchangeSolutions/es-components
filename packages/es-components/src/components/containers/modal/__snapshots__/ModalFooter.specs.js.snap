// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`modal footer component renders as expected 1`] = `
<div
  className="footer sc-bdVaJa dtJhjk"
>
  This is the footer.
</div>
`;
